src/convertToObject.js

```diff
@@ -7,6 +7,21 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  return sourceString
+    .split(';')
+    .map((item) => item.trim())
+    .filter((item) => item.includes(':'))
+    .reduce((obj, item) => {
+      const [property, ...value] = item.split(':');
+      const trimProperty = property.trim();
+      const trimValue = value.join(':').trim();
+
+      if (trimProperty && trimValue) {
+        obj[trimProperty] = trimValue;
+      }
+
+      return obj;
+    }, {});
 }
 
 module.exports = convertToObject;
```

Comment:
Exporting the `convertToObject` function using `module.exports` is correct and allows it to be imported and used in other modules.